framework/graphics/fps_info.cpp

```diff
@@ -54,16 +56,26 @@ WriteFpsToConsole(const char* prefix, uint64_t start_frame, uint64_t end_frame,
                            end_frame);
 }
 
-FpsInfo::FpsInfo(uint64_t measurement_start_frame,
-                 uint64_t measurement_end_frame,
-                 bool     has_measurement_range,
-                 bool     quit_after_range,
-                 bool     flush_measurement_range) :
+FpsInfo::FpsInfo(uint64_t           measurement_start_frame,
+                 uint64_t           measurement_end_frame,
+                 bool               has_measurement_range,
+                 bool               quit_after_range,
+                 bool               flush_measurement_range,
+                 const std::string& measurement_file_name) :
```

Comment:
You could make this a `const string_view` for flexibility if you wanted but this is not a requirement.

Reply:
Done